src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -982,6 +982,51 @@ public static IEnumerable<object[]> GetAsync_IPBasedUri_Success_MemberData()
             });
         }
 
+        [Theory]
```

Comment:
Added OuterLoop (since most of the tests already have it). Please let me know if it should better be removed.